mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -3238,6 +3241,10 @@ private void showBrowserSearch() {
         getWindow().setBackgroundDrawableResource(android.R.color.white);
     }
 
+    private void showCliqzSearch() {
```

Comment:
@spacifici I would add this clsoe to hideCliqzSearch and using the comment above. Less change surface - less headaches during merge